.github/workflows/ci.yaml

```diff
@@ -228,6 +228,10 @@ jobs:
       - name: Install Maven modules
         run: |
           mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
+      - name: Install logging module # this module is not part of root project.
```

Comment:
Why do we want to install logging module in the main CI?

Reply:
Showcase needs to add the logging module as a dependency.
https://github.com/googleapis/sdk-platform-java/pull/3693/files#diff-3c3eb08864bc257af6721c1c900ec0cbb8568e2045b8aadc3330756a97b6ef1eR298